app/views/idv/cac/choose_method.html.erb

```diff
@@ -1,24 +1,45 @@
 <%= render 'shared/maintenance_window_alert' do %>
+  <style>
+    .flex-container {
+        display: flex;
+    }
+    @media (max-width: 30em) {
+        .flex-container {
+            flex-direction: column;
+        }
+    }
+  </style>
   <h1 class="h3 mt0 mb1"><%= t('cac_proofing.headings.choose_method') %></h1>
-  <%= t('cac_proofing.info.choose_method') %>
+  <p><%= t('cac_proofing.info.choose_method') %></p>
+  <p><%= t('cac_proofing.info.disclaimer') %></p>
+  <div class="flex-container mt3 mb3">
+    <%= image_tag asset_url('PIV-CAC-round.svg'), width: 120, height: 116, class: 'mr3 mb2'%>
+    <div>
+      <div id="recommended-tag"
+           class='usa-tag caps text-ink bg-primary-lighter margin-top-1 mb2 display-inline-block'>
+        <%= t('doc_auth.info.tag') %>
+      </div>
+      <p><%= t('cac_proofing.info.piv_cac_description') %></p>
+      <%= validated_form_for :cac,
+                             url: url_for,
+                             method: 'put',
+                             html: { autocomplete: 'off', role: 'form', class: 'mt2' } do |f| %>
+        <button class="btn btn-primary" type="submit" style="text-align:center;">
```

Comment:
Is the inline styling necessary? I don't notice any difference without it.

```suggestion
        <button class="btn btn-primary" type="submit">
```

Reply:
In this case, it doesn't do anything in english but it's for spanish and french. The text is much longer and flows onto two lines  and needs the centering to look a little less terrible in that case.

Reply:
Interesting. I'd wonder if it's something which should be default behavior of these sorts of primary action buttons, but given that we're already going to need to do some migration to USWDS, probably not worth worrying too much over now.

Reply:
Though, as with our earlier convo, I did fix it to use USWDS class rather than an inline style.